README.md

```diff
@@ -30,16 +30,20 @@ When using SFTP, you should not use the `--file` option, instead you need to pro
 
 ### CSV Format
 
-Column 1 will be used as `SKU` identifier, whereas column 2 will be used as `quantity`.
-An example:
+A simple example:
 ```
-sku,quantity
-foo,9
-bar,-1
-SKU-123,42
+sku,quantityOnStock,restockableInDays,supplyChannel,expectedDelivery
```

Comment:
FYI: since we're introducing a breaking change with `quantityOnStock`, we need to bump a major version + document this in the release notes.

We can also try to make it backwards compatible:
- `quantityOnStock` is the new default
- if client uses `quantity`, we map it internally to `quantityOnStock` + we warn the client that this is deprecated